src/components/product-tile/product-price/ProductPrice.tsx

```diff
@@ -7,20 +7,25 @@ const ProductPriceContainer = styled.h4`
   margin-bottom: ${({ theme }) => theme.sizes.spacing.md};
 `;
 
+type PriceLocale = 'en' | 'fr' | 'pt-PT';
```

Comment:
should we have all the locales like language-country then? like en-GB

Reply:
Can do!